### DISCUSSION TITLE
Flight can't stop when reach destination.

### DISCUSSION THREAD

**Discussion by anhduong89** [TUDelft-CNS-ATM/bluesky#484] [General]
Hi,
I started using Bluesky recently. I want to set up a simple scenario that I only need a flight start from origin and stop at destination. Below is my scenario(which I took from nl22-nmsu and we all know it worked):

00:00:00.00> trail on
00:00:00.00> cre k1 b737 53 -30 0 3000 245
00:00:00.00> orig k1 53 -30
00:00:00.00> dest k1 54 -29
00:00:00.00> pan k1

However, when the flight reach the destination, the speed of flight doesn't reduce to 0, and it keeps stuck at 177 speed. I run Bluesky in macOS system.

I don't know what I did wrong. Could you guys help me with that problem? I really appreciate.

Thank you.

**Comment by ProfHoekstra** [comment]
Looks like the performance model is interfering and does not recognize the on ground situation. Are you using OpenAP? Could you try it with BADA?

Best regards/Met vriendelijke groeten,
    Jacco Hoekstra


On 13 Oct 2023, at 05:41, anhduong89 ***@***.***> wrote:

﻿

Hi,
I started using Bluesky recently. I want to set up a simple scenario that I only need a flight start from origin and stop at destination. Below is my scenario(which I took from nl22-nmsu and we all know it worked):

00:00:00.00> trail on
00:00:00.00> cre k1 b737 53 -30 0 3000 245
00:00:00.00> orig k1 53 -30
00:00:00.00> dest k1 54 -29
00:00:00.00> pan k1

However, when the flight reach the destination, the speed of flight doesn't reduce to 0, and it keeps stuck at 177 speed. I run Bluesky in macOS system.

I don't know what I did wrong. Could you guys help me with that problem? I really appreciate.

Thank you.

**Reply by anhduong89** [reply]
Hi Prof.Hoekstra, 

Thank you for the instruction. I'm in the progress of obtaining BADA license and will update to you the result as soon as possible.

Yes currently I'm using openAP model. It worked on my colleague's computer but didn't work on mine(we are in different location, and I did try Bluesky on 3 different machines 2 macOS, 1 windows OS). I tried to install Bluesky in the same way(anaconda) and same OS(windows) as my friend did but it still doesn't work.
